common/app/views/fragments/footer.scala.html

```diff
@@ -94,6 +94,9 @@
                     contact us</a></li>
             }
 
+        <li class="colophon__item footer__feedback--hide js-tech-feedback"><a data-link-name="uk : footer : tech feedback">report technical issue</a></li>
```

Comment:
How about using the switch here? If the switch is disabled you can avoid sending this.
It would make the line `link.removeClass('footer__feedback--hide');` unnecessary in `tech-feedback.js`.